cloudigrade/api/error_codes.py

```diff
@@ -179,7 +180,29 @@ def notify(self, account_number, application_id, error_message=None):
     _("Failed to enable Cloud Account %(cloud_account_id)s: " "because %(exception)s"),
     _(
         "Could not enable cloud metering. You've reached the AWS CloudTrail "
-        "limit for your account. Please either increase the CloudTrail limit, or "
-        "delete an existing CloudTrail. Error code %(error_code)s."
+        "limit for your account. Please ensure you have not reached the CloudTrail "
+        "limit for the your account, and try again. Error code %(error_code)s."
+    ),
+)
+
+# Misconfigured role
+CG3002 = CloudigradeError(
+    "CG3002",
+    _("Failed to enable Cloud Account %(cloud_account_id)s: " "because %(exception)s"),
+    _(
+        "Could not enable cloud metering. An AccessDenied was received when using  "
```

Comment:
also maybe "An AccessDenied error" instead of "An AccessDenied" reads more human